HATE/Functionality.cs

```diff
@@ -12,7 +12,7 @@ public static bool ShuffleAudio_Func(UndertaleData data, Random random, float ch
         return Shuffle.ShuffleChunk(data.FORM.Chunks.GetValueOrDefault("SOND"), data, random, chance, logstream, _friskMode)
                && Shuffle.ShuffleChunk(data.FORM.Chunks.GetValueOrDefault("STRG"), data, random, chance, logstream, _friskMode,
                                        Shuffle.ComplexShuffle(Shuffle.ShuffleAudio2_Shuffler));
-        //&& Shuffle.ShuffleChunk(data.EmbeddedAudio, data, random, chance, logstream);               
+        //&& Shuffle.ShuffleChunk(data.EmbeddedAudio, data, random, chance, logStream);               
```

Comment:
Do you want to rename the variable for the whole function? Because you didn't

Reply:
that came accidentally lol